### DISCUSSION TITLE
Enlarge and colorize bullet mark

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#3207] [Q&A] [answered]
Consider the following code, which produces the output below. Is it possible to enlarge the black circles and give them another color, say, red?

```
- First item
- Second item
- Third item
```

![image](https://github.com/typst/typst/assets/75945439/ba6082ec-7f15-4a71-8c88-c02f49dfb5cf)

**Comment by xkevio** [comment, marked as answer]
You can change the `marker` parameter of `list` with a set-rule since it takes any form of content. You might have to adjust the baseline depending on how much bigger you make the markers to keep them horizontally aligned.

```typ
#set list(marker: text(20pt, red, [•], baseline: -5pt))

- First item
- Second item
- Third item
```

![image](https://github.com/typst/typst/assets/13004777/1d0ed5dd-6a3c-42a1-b3bd-a24bcdb124cc)

**Reply by psads-git** [reply]
Great! Thanks, @xkevio!

**Reply by gerardolujanhdz** [reply]
Thanks for that @xkevio! really helpful :D
